pydm/application.py

```diff
@@ -132,13 +132,27 @@ def load_ui_file(self, uifile, macros=None):
       f = uifile
     return uic.loadUi(f)
     
+  def __sanity_check_pyqt(self, cls):
+    def check(attr):
```

Comment:
Doing a quick test I noticed that it is not too heavy of a burden to add it and it will reduce a good amount of problems in the future...
Here is the test result:
```
1000000 loops, best of 3: 1.65 µs per loop
```